R/errors.R

```diff
@@ -284,8 +284,8 @@ accuracy.default <- function(object, x, test=NULL, d=NULL, D=NULL, f = NULL, ...
         D <- as.numeric(frequency(object$mean) > 1)
       }
       else {
-        d <- as.numeric(frequency(object$x) == 1)
-        D <- as.numeric(frequency(object$x) > 1)
+        d <- as.numeric(frequency(object$xlevels) == 1)
```

Comment:
I think object$x is appropriate here.
Perhaps use object[["x"]] instead.

Reply:
Hmm, this did raise partial match warnings, but the `[[` accessor also works!

Reply:
`$xlevels` is an element of the `tslm()` output, but (most) models will store the original series as `$x`. This code is looking for the data's frequency, and so `x` is the element we want.